src/capwap_tools.erl

```diff
@@ -31,6 +32,25 @@ format_mac(<<A:8, B:8, C:8, D:8, E:8, F:8>>) ->
 format_mac(MAC) ->
     flat_format("~w", MAC).
 
+mac_to_hex(MAC) when is_binary(MAC)->
```

Comment:
for to complicate, here is a much simple version: https://github.com/travelping/capwap/blob/1af16c4c354c797113911a236245ccb7c3f92713/src/capwap.escript#L127